data_ingestion/02_Ingest_data_with_Athena_v1.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>I would explicitly state here, you are referring to the execution role used to create this notebook instance. It might also be helpful to print out the role name after a call to <code>sagemaker.get_execution_role</code> so the customer knows which role to attach this policy to in the console. </p><p> </p><p>Grammer:</p><blockquote>You can do this through IAM client as below,</blockquote><p>Update to</p><blockquote>You can do this through an IAM client as shown below,</blockquote><p> </p><p>Update the <strong>Note</strong> to:</p><blockquote>You need <a href="https://console.aws.amazon.com/iam/home?region=us-west-2#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FIAMFullAccess" target="_blank">IAMFullAccess</a> to attach policies to the role.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>